nservicebus/msmq/distributor/configuration_distributor_core_[,4).partial.md

```diff
@@ -0,0 +1,7 @@
+To start the endpoint as a Distributor run the host from the command line, as follows:
+
+```dos
+NServiceBus.Host.exe NServiceBus.Distributor
+```
+
+The NServiceBus.Distributor profile instructs the NServiceBus framework to start a Distributor on this endpoint, waiting for workers to enlist to it. Unlike the NServiceBus.Master profile, the NServiceBus.Distributor profile does not execute a Worker on its node.
```

Comment:
> Unlike the NServiceBus.Master profile, the NServiceBus.Distributor profile does not execute a Worker on its node

I don't understand this sentence, what does it mean that NServiceBus.Master profile "executes a Worker on its node"? Do we mention/explain this profile elsewhere? 

Also it's not clear to me why we mention NServiceBus.Master profile here, since in the snippet we cover only NServiceBus.Distributor profile. Can we explain when that profile is used earlier (I see the explanation is in the partial below)?